### ISSUE TITLE
Which aspects of the guidance collected in a recent paper on chatbot accessibility should be included in tis document?

### ISSUE THREAD

**Issue by jasonjgw** [w3c/apa#262]
The following paper, recently published on the subject of chatbot accessibility, collects guidance found in relevant literature - including speech interface as well as chatbot-related sources.

Stanley, J., Brink, R. T., Valiton, A., Bostic, T., & Scollan, B. (2022). Chatbot Accessibility Guidance: A Review and Way Forward. In Proceedings of Sixth International Congress on Information and Communication Technology (pp. 919-942). Springer, Singapore.

Which of the issues identified in this paper should be covered in our document, and how should they be addressed?

Whereas the paper cited above lists all of the suggestions found in the literature, our document aims to be based on well founded judgments about what user needs are significant to accessibility, and how they can best be met in the design of natural language interfaces. Thus, we do not aim to collect all of the design strategies that have been suggested, but rather to develop a carefully considered set of user needs and associated requirements.

Nevertheless, there are issues raised in the paper that should be considered for inclusion.

**Comment by jasonjgw** [comment]
Mitre Corporation guidance on chatbot accessibility has now been published.
https://mitre.github.io/chatbot-accessibility-playbook/
